### PR TITLE
feat: onboarding adjust source

### PR DESCRIPTION
**Pull request by shrouti1507**
## What are the changes introduced in this PR?

Write a brief explainer on your code changes.

## What is the related Linear task?

Resolves INT- 1826

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3395/test-report.html

File: src/v0/sources/adjust/transform.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+const processEvent = (inputEvent) => {
+  const message = new Message(`Adjust`);
+  const event = lodash.cloneDeep(inputEvent);
+  const formattedPayload = formatProperties(event);
+  // event type is always track
+  const eventType = 'track';
+  message.setEventType(eventType);
+  // message.setEventName(event.event_name);
```
Comment (anantjain45823) [inline src/v0/sources/adjust/transform.js:34]: Can we remove this comment

Comment (shrouti1507) [reply src/v0/sources/adjust/transform.js]: done

**Review by anantjain45823 — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

File: test/integrations/sources/adjust/data.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import utils from '../../../../src/v0/util';
+
+const defaultMockFns = () => {
+  jest.spyOn(utils, 'generateUUID').mockReturnValue('97fcd7b2-cc24-47d7-b776-057b7b199513');
+};
+
+export const data = [
+  {
+    name: 'adjust',
```
Comment (utsabc) [inline test/integrations/sources/adjust/data.ts:9]: All test cases should have id property

**Review by utsabc — APPROVED** [review]

File: test/integrations/sources/adjust/data.ts (hunk 2 above)
Comment (shrouti1507) [reply test/integrations/sources/adjust/data.ts:9]: will add this in next release as it is a minor issue.

**Pull request merged**